lib/registry.js

```diff
@@ -445,6 +432,3 @@ Object.defineProperty(registry, 'DataModel', {
     return this.PersistedModel;
   }
 });
-
-// Set the default model base class. This is done after the Model class is defined.
-registry.modelBuilder.defaultModelBaseClass = registry.Model;
```

Comment:
This code was removed with not replacement. Are you sure it's not a breaking change?